### PR TITLE
add nephrops plots to main

### PR DESCRIPTION
**Pull request by lucalamoni**

### CODE REVIEW

File: App/server.r [hunk 1]
```diff
@@ -326,25 +317,27 @@ onclick("library_advice_link2", runjs(paste0("window.open('", advice_doi(),"', '
 
 ##### Advice view info
 advice_view_info <- reactive({
-  get_Advice_View_info(query$stockkeylabel, query$year)
+  # get_Advice_View_info(query$stockkeylabel, query$year)
```
Comment (Neilmagi) [inline App/server.r]: remove duplicate?

File: App/server.r [hunk 2]
```diff
@@ -326,25 +317,27 @@ onclick("library_advice_link2", runjs(paste0("window.open('", advice_doi(),"', '
...
+  get_advice_view_info(query$stockkeylabel, query$year)
 }) 
 
 
 ##### Advice view info previous year
 advice_view_info_previous_year <- eventReactive(req(query$stockkeylabel,query$year), {
-  get_Advice_View_info(query$stockkeylabel, query$year-1)
+  # get_Advice_View_info(query$stockkeylabel, query$year-1)
```
Comment (Neilmagi) [inline App/server.r]: duplicate?

File: App/server.r (hunk 1 above)
Comment (lucalamoni) [reply App/server.r]: yep, I'll remove it and re-commit.

File: App/server.r (hunk 2 above)
Comment (lucalamoni) [reply App/server.r]: same as above

**Review by Neilmagi — APPROVED** [review]

**Review by Neilmagi — APPROVED** [review]

**Pull request merged**